VagrantProvisionVars.sh

```diff
@@ -1,9 +1,6 @@
 #!/usr/bin/env bash
 
-export JDK_VERSION=1.8.0_161
-export JDK_URL=http://download.oracle.com/otn-pub/java/jdk/8u161-b12/2f38c3b165be4555a1fa6e98c45e0808/jdk-8u161-linux-x64.tar.gz
-export JDK_TAR=jdk-8u161-linux-x64.tar.gz
-export JDK_MD5=99051574a0d90871ed24a91a5d321ed2
```

Comment:
I couldn't find where this is used anywhere, but genericized the JDK_VERSION and removed hardcoded `1.8.0` elsewhere.